### ISSUE TITLE
Sequence Diagram #1: Login

### ISSUE THREAD

**Issue by oguzhekim** [bounswe/bounswe2024group7#32]
We need to create sequence diagrams based on the use case diagram mentioned in [this issue](https://github.com/bounswe/bounswe2024group7/issues/31). First sequence diagram will be about user trying to login.

**Comment by MertCengiz** [comment]
The first version of the First Sequence Diagram related to logging in is avaliable [here](https://github.com/bounswe/bounswe2024group7/blob/main/Sequence%20Diagram-1-Logging-In.mmd). Review needed and highly recommended.

**Comment by oguzhekim** [comment]
I have revised the login sequence diagram according to the feedback from TA. These are the changes I have made:
* Removed GuestUser and added Interface and System instead.
* Showed the communication between Interface-System and System-Database.
* Wrote the communication between the actor and interface in an abstract manner.

The new diagram is available [here](https://github.com/bounswe/bounswe2024group7/wiki/Sequence-Diagrams), along with the other sequence diagrams.
